flow/src/main/java/com/sequenceiq/flow/service/FlowService.java

```diff
@@ -195,6 +201,29 @@ public FlowCheckResponse getFlowState(String flowId) {
         flowCheckResponse.setFlowId(flowId);
         flowCheckResponse.setHasActiveFlow(!completed("Flow", flowId, List.of(), allByFlowIdOrderByCreatedDesc));
         flowCheckResponse.setLatestFlowFinalizedAndFailed(isFlowInFailedState(allByFlowIdOrderByCreatedDesc, failHandledEvents));
+        // Get the latest flow information that will be used for error logging.
```

Comment:
Please omit the comments describing the code. Use appropriate method and variable names.

Reply:
Removed the comments. TYSM!